<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Aipage</class>
 <widget class="QMainWindow" name="Aipage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background: qlineargradient(
        spread: pad,
        x1: 0, y1: 0,
        x2: 1, y2: 1,
        stop: 0 rgba(0, 34, 133, 180),
        stop: 1 rgba(255, 255, 255, 80)
    );
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_p">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    border: none;
}
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="start_game_ai">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: none;
    color: white;
    background-color: rgba(0, 120, 255, 180); /* ÿ£ÿ≤ÿ±ŸÇ ÿ¥ŸÅÿßŸÅ */
    font: bold 11pt &quot;Arial&quot;;
    border-radius: 20px;
    padding: 8px;
}

QPushButton:hover {
    background-color: rgba(30, 144, 255, 220); /* ÿ£ÿ≤ÿ±ŸÇ ÿ£ŸÅÿ™ÿ≠ ÿπŸÜÿØ ÿßŸÑŸÖÿ±Ÿàÿ± */
    font: bold 12pt &quot;Arial&quot;;
}

QPushButton:pressed {
    background-color: rgba(0, 80, 180, 255); /* ÿ£ÿ≤ÿ±ŸÇ ÿØÿßŸÉŸÜ ÿπŸÜÿØ ÿßŸÑÿ∂ÿ∫ÿ∑ */
    font: bold 12pt &quot;Arial&quot;;
}

QPushButton:disabled {
    background-color: rgba(150, 150, 150, 100); /* ÿ±ŸÖÿßÿØŸä ŸÅÿßÿ™ÿ≠ ÿ¥ŸÅÿßŸÅ */
    color: rgba(255, 255, 255, 180); /* ÿ£ÿ®Ÿäÿ∂ ÿ®ÿßŸáÿ™ */
    font: bold 11pt &quot;Arial&quot;;
}
</string>
     </property>
     <property name="text">
      <string>Start Game</string>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="turn_box_ai">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #e0f7fa, stop:1 #b2ebf2); /* gradient blue */
    color: #004d40; /* deep teal text */
    border: 2px solid #4dd0e1;
    border-radius: 12px;
    padding: 8px 12px;
    font: bold 12pt &quot;Segoe UI&quot;;
}

QComboBox:hover {
    background-color: #b2ebf2;
    border: 2px solid #00acc1;
}

QComboBox::drop-down {
    border: none;
    width: 25px;
    background: transparent;
}

QComboBox::down-arrow {
    image: url(:/icons/arrow_down_dark.png); 
    width: 14px;
    height: 14px;
}

QComboBox QAbstractItemView {
    background-color: #ffffff;
    color: #004d40;
    border: 1px solid #4dd0e1;
    selection-background-color: #b2dfdb;
    font: 11pt &quot;Segoe UI&quot;;
}
</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <property name="placeholderText">
     <string>      ‚ùå or ‚≠ï</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ai">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>-10</y>
      <width>501</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    background-color: qlineargradient(
        x1: 0, y1: 0,
        x2: 1, y2: 1,
        stop: 0 rgba(30, 60, 114, 150),   /* Dark Blue with transparency */
        stop: 1 rgba(42, 82, 152, 150)    /* Medium Blue with transparency */
    );
    color: white;
    font: bold 17pt &quot;Segoe UI&quot;;
    padding: 14px;
    border-radius: 20px;
    border: 2px solid rgba(255, 255, 255, 180); /* Semi-transparent white border */
    qproperty-alignment: AlignCenter;
}

QLabel:hover {
    background-color: qlineargradient(
        x1: 1, y1: 0,
        x2: 0, y2: 1,
        stop: 0 rgba(58, 28, 113, 180),   /* Dark purple */
        stop: 1 rgba(215, 109, 119, 180)  /* Pink with transparency */
    );
    color: #FFEB3B;
    border: 2px solid #FFEB3B;
}
</string>
    </property>
    <property name="text">
     <string>Ai Difficulty ü§ñ and Starting Turn</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_tomain_ai">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>220</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: none;
    color: white;
    background-color: rgba(0, 120, 255, 180); /* ÿ£ÿ≤ÿ±ŸÇ ÿ¥ŸÅÿßŸÅ */
    font: bold 11pt &quot;Arial&quot;;
    border-radius: 20px;
    padding: 8px;
}

QPushButton:hover {
    background-color: rgba(30, 144, 255, 220); /* ÿ£ÿ≤ÿ±ŸÇ ÿ£ŸÅÿ™ÿ≠ ÿπŸÜÿØ ÿßŸÑŸÖÿ±Ÿàÿ± */
    font: bold 12pt &quot;Arial&quot;;
}

QPushButton:pressed {
    background-color: rgba(0, 80, 180, 255); /* ÿ£ÿ≤ÿ±ŸÇ ÿØÿßŸÉŸÜ ÿπŸÜÿØ ÿßŸÑÿ∂ÿ∫ÿ∑ */
    font: bold 12pt &quot;Arial&quot;;
}
</string>
    </property>
    <property name="text">
     <string>Main Menu</string>
    </property>
   </widget>
   <widget class="QComboBox" name="level_ai">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #e0f7fa, stop:1 #b2ebf2); /* gradient blue */
    color: #004d40; /* deep teal text */
    border: 2px solid #4dd0e1;
    border-radius: 12px;
    padding: 8px 12px;
    font: bold 12pt &quot;Segoe UI&quot;;
}

QComboBox:hover {
    background-color: #b2ebf2;
    border: 2px solid #00acc1;
}

QComboBox::drop-down {
    border: none;
    width: 25px;
    background: transparent;
}

QComboBox::down-arrow {
    image: url(:/icons/arrow_down_dark.png); 
    width: 14px;
    height: 14px;
}

QComboBox QAbstractItemView {
    background-color: #ffffff;
    color: #004d40;
    border: 1px solid #4dd0e1;
    selection-background-color: #b2dfdb;
    font: 11pt &quot;Segoe UI&quot;;
}
</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <property name="placeholderText">
     <string>   ü§ñ Ai Level</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>434</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
